File: Solutions/3408/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        String[] s = new String[n];
        for(int i = 0; i < n; i++) {
            s[i] = scan.next();
        }
        for(int i = n-1; i >= 0; i--) {
            System.out.printf("%s ", s[i]);
        }
    }
}

File: Solutions/3408/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;


void printVerbsInSequence(int numOfVerbs)
{
	if(numOfVerbs == 0)
	{
		return;
	}
	char str[100];
	cin >> str;
	printVerbsInSequence(numOfVerbs - 1);
	cout << str << " ";
}

int main()
{
	int n;
	cin >> n;
	printVerbsInSequence(n);
}
